rego-templates/common/common.rego

```diff
@@ -29,3 +29,17 @@ with_default(obj, prop, default_value) = default_value{
 with_default(obj, prop, default_value) = obj[prop]{
  obj[prop]
 }
+
+flat_one_dim_array(a) = o {
```

Comment:
What is purpose of this function? It looks like `noop` to me: https://play.openpolicyagent.org/p/MO444fmD5Y